web/src/stream_popover.ts

```diff
@@ -470,9 +471,24 @@ export async function build_move_topic_to_stream_popover(
         // disabled in case when user does not have permission to edit
         // topic and thus submit button is disabled if stream is also
         // not changed.
-        util.the($<HTMLButtonElement>("#move_topic_modal button.dialog_submit_button")).disabled =
+        let is_disabled = false;
+        if (old_topic_name === "(no topic)" && new_topic_name === "") {
```

Comment:
Think this would also require calling `trim()`. Would be just good to define `new_topic_name = new_topic_name.trim()` at the top.